common/kafka/kafka_watcher.cpp

```diff
@@ -223,6 +223,10 @@ uint32_t KafkaWatcher::ConsumeUpToNow(const kafka::KafkaConsumer& consumer) {
       // happens when consumer hasn't got any message from the broker within the
       // timeout. We should retry in this case.
     } else {
+      ++err_count_;
```

Comment:
can we use fetch_add() instead of ++ to make it more explicit that err_count_ is a std::atomic?

Reply:
thanks for suggestion. done